code/WorkInProgress/telescope/telescopeMisc.dm

```diff
@@ -23,6 +24,9 @@ var/list/magnet_locations = list()
 		return attack_hand(user)
 
 	attack_hand(mob/user as mob)
+		if(ui_interact(user))
+			return
+
```

Comment:
I would think that the rest of this proc is probably not needed now? If it is, why?

Reply:
If Tgui were to fail it would use old UI. Can remove. 🙂

Reply:
Removing!